src/olympia/devhub/tests/test_views_validation.py

```diff
@@ -404,166 +396,3 @@ def test_upload_addon_version(self):
-            "messages": [],
-            "ending_tier": 5,
-            "metadata": {
-                "contains_binary_extension": True,
-                "version": "1.0",
-                "name": "gK0Bes Bot",
-                "id": "gkobes@gkobes"
-            }
-        })
-        response = self.client.post(
-            reverse('devhub.json_file_validation',
-                    args=[self.addon.slug, self.file.id]), follow=True)
-        assert response.status_code == 200
-        data = json.loads(response.content)
-        assert not data['validation']['errors']
-        assert data['validation']['ending_tier'] == 5
-
-    @mock.patch('olympia.devhub.tasks.validate_file_path')
-    def test_validator_sets_binary_flag_for_content(
-            self, validate_file_path_mock):
-        validate_file_path_mock.return_value = json.dumps({
-            "errors": 0,
-            "success": True,
-            "warnings": 0,
-            "notices": 0,
-            "message_tree": {},
-            "messages": [],
-            "metadata": {
-                "contains_binary_content": True,
-                "version": "1.0",
-                "name": "gK0Bes Bot",
-                "id": "gkobes@gkobes"
-            }
-        })
-        assert not self.addon.binary
-        response = self.client.post(
-            reverse('devhub.json_file_validation',
-                    args=[self.addon.slug, self.file.id]), follow=True)
-        assert response.status_code == 200
-        data = json.loads(response.content)
-        assert not data['validation']['errors']
-        addon = Addon.objects.get(pk=self.addon.id)
-        assert addon.binary
-
-    @mock.patch('olympia.devhub.tasks.validate_file_path')
-    def test_linkify_validation_messages(self, validate_file_path_mock):
-        validate_file_path_mock.return_value = json.dumps({
-            "errors": 0,
-            "success": True,
-            "warnings": 1,
-            "notices": 0,
-            "message_tree": {},
-            "messages": [{
-                "context": ["<code>", None],
-                "description": [
-                    "Something something, see https://bugzilla.mozilla.org/"],
-                "column": 0,
-                "line": 1,
-                "file": "chrome/content/down.html",
-                "tier": 2,
-                "message": "Some warning",
-                "type": "warning",
-                "id": [],
-                "uid": "bb9948b604b111e09dfdc42c0301fe38"
-            }],
-            "metadata": {}
-        })
-        response = self.client.post(
-            reverse('devhub.json_file_validation',
-                    args=[self.addon.slug, self.file.id]), follow=True)
-        assert response.status_code == 200
-        data = json.loads(response.content)
-        doc = pq(data['validation']['messages'][0]['description'][0])
-        assert doc('a').text() == 'https://bugzilla.mozilla.org/'
-
-    def test_opensearch_validation(self):
```

Comment:
Should we keep this test case even if we are going to remove opensearch support soon?

Reply:
Given that we are removing it soon and it was in the wrong place I wasn't really motivated to keep it...